pkg/observ/stats.go

```diff
@@ -0,0 +1,81 @@
+package observ
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gobuffalo/buffalo"
+	"github.com/gomods/athens/pkg/errors"
+	"go.opencensus.io/exporter/prometheus"
+	"go.opencensus.io/plugin/ochttp"
+	"go.opencensus.io/stats/view"
+)
+
+// RegisterStatsExporter determines the type of StatsExporter service for exporting stats from Opencensus
+// Currently it supports: prometheus
+func RegisterStatsExporter(app *buffalo.App, statsExporter, service string) (func(), error) {
+	const op errors.Op = "RegisterStatsExporter"
+
+	switch statsExporter {
+	case "prometheus":
+		if _, err := registerPrometheusExporter(app, service); err != nil {
+			return nil, errors.E(op, err)
+		}
+	case "":
+		return nil, errors.E(op, "StatsExporter not specified. Stats won't be collected")
+	default:
+		return nil, errors.E(op, fmt.Sprintf("StatsExporter %s not supported. Please open PR or an issue at github.com/gomods/athens", statsExporter))
+	}
+
+	if err := registerViews(); err != nil {
+		return nil, errors.E(op, err)
+	}
+
+	return func() {}, nil
```

Comment:
Currently `func() {}` it's not needed but when we would add StackDriver or DataDog exporter it will be needed and by having that already here it will be easy to implement.

Reply:
can you then add a comment there so we know why it is there.
i'm not a big fan of future code and what if we will need it approach.

Reply:
Added, also removed that func() from `registerPrometheusExporter` method